MultiTable/src/RealtimeChangeHandler.js

```diff
@@ -44,6 +44,7 @@ function onSampleUpdate(data, change) {
 function onSubjectAdd(data, subject) {
```

Comment:
I don't know how to install in prod, or if I have access. You showed me briefly when I started but not in depth.